api/resources_portal/test/test_data.json

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
This data is for local development. While we'll use it to test things locally, I think it'd be good to differentiate it from data that will be used for unit/end-to-end tests. Can we call this dev_data instead?

Also, this file ended up being a big ol' whoppin file. Can we break it up a bit? Maybe something like:
```
dev_data/
  auth.json
  users.json
  materials.json
  etc
```

We don't need a different file for every single resource (I'm thinking about all the relationship tables), but I think grouping them up a bit will make it easier to work with when we want to update or extend it.

Reply:
Sounds good. Should I go ahead and do that now, then push again? Or should I wait for the other reviewers?